docs/docusaurus.config.js

```diff
@@ -0,0 +1,76 @@
+// @ts-check
+// Note: type annotations allow type checking and IDEs autocompletion
+
+const lightCodeTheme = require('prism-react-renderer/themes/github');
+const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title: 'Eraser Docs',
+  url: 'https://azure.github.io/eraser',
```

Comment:
i remember url and baseUrl were tricky, make sure to test and deploy on your branch